[BUG]

### Title

Invalid SSH key crashing continuously

### Platform

App - Linux

### Server Installation Method

Docker

### Version

1.11.0

### Troubleshooting

- [x] I have examined logs and tried to find the issue
- [x] I have reviewed opened and closed issues
- [x] I have tried restarting the application
- [x] I have read Known Issues (https://github.com/Termix-SSH/Support/issues/296)

### The Problem

It appears that when I added an SSH key into my credentials, the formatting was invalid for the key pair. Now when the docker container starts and sees the key for that server is invalid, Termix will crash and restart indefinitely and I'm unable to get into the UI long enough to remove the server or token causing the problem. Have tried to access the database to edit it and remove the server causing this, but haven't been able to access the encrypted database to remove it. Would prefer not to have to nuke the database and start over.

Log files are showing this output repeatedly
`[termix] 2026-02-09T20:10:59.667502139Z [8:10:59 PM] [INFO] [📡] Found 0 autostart hosts and 17 total hosts for endpointHost resolution
[termix] 2026-02-09T20:11:01.903318737Z [8:11:01 PM] [WARN] [🗄️] Session not found during JWT verification [op:jwt_verify_session_not_found,user:RW2GVM8qAEB9tt3PGWErE,session:ED-qSZtwbsyf6VEA9nkMp]
[termix] 2026-02-09T20:11:01.905223174Z [8:11:01 PM] [WARN] [🗄️] Session not found during JWT verification [op:jwt_verify_session_not_found,user:RW2GVM8qAEB9tt3PGWErE,session:ED-qSZtwbsyf6VEA9nkMp]
[termix] 2026-02-09T20:11:01.906228047Z [8:11:01 PM] [WARN] [🗄️] Session not found during JWT verification [op:jwt_verify_session_not_found,user:RW2GVM8qAEB9tt3PGWErE,session:ED-qSZtwbsyf6VEA9nkMp]
[termix] 2026-02-09T20:11:01.907479614Z [8:11:01 PM] [WARN] [🗄️] Session not found during JWT verification [op:jwt_verify_session_not_found,user:RW2GVM8qAEB9tt3PGWErE,session:ED-qSZtwbsyf6VEA9nkMp]
[termix] 2026-02-09T20:11:01.910654284Z [8:11:01 PM] [WARN] [🗄️] Session not found during JWT verification [op:jwt_verify_session_not_found,user:RW2GVM8qAEB9tt3PGWErE,session:ED-qSZtwbsyf6VEA9nkMp]
[termix] 2026-02-09T20:11:01.911572063Z [8:11:01 PM] [WARN] [🗄️] Session not found during JWT verification [op:jwt_verify_session_not_found,user:RW2GVM8qAEB9tt3PGWErE,session:ED-qSZtwbsyf6VEA9nkMp]
[termix] 2026-02-09T20:12:45.038616316Z [8:12:45 PM] [SUCCESS] [🔐] User logged in successfully: ddt [op:user_login_success,user:RW2GVM8qAEB9tt3PGWErE]
[termix] 2026-02-09T20:12:47.754881095Z [8:12:47 PM] [ERROR] [📊] Failed to collect metrics for host
[termix] 2026-02-09T20:12:47.755723431Z {
[termix] 2026-02-09T20:12:47.755741565Z   operation: 'metrics_poll_failed',
[termix] 2026-02-09T20:12:47.755747556Z   hostId: 4,
[termix] 2026-02-09T20:12:47.755749259Z   hostName: 'Frigate',
[termix] 2026-02-09T20:12:47.755750492Z   error: 'read ECONNRESET',
[termix] 2026-02-09T20:12:47.755751423Z   backoff: 'Retry in 30s (attempt 1/5)'
[termix] 2026-02-09T20:12:47.755762434Z }
[termix] 2026-02-09T20:12:47.766433030Z [8:12:47 PM] [ERROR] [📊] SSH key format error for host X.X.X.X: Invalid private key format
[termix] 2026-02-09T20:12:47.766448830Z [8:12:47 PM] [ERROR] [🚀] Unhandled promise rejection [op:error_handling]
[termix] 2026-02-09T20:12:47.766450483Z Error: Invalid SSH key format for host X.X.X.X
[termix] 2026-02-09T20:12:47.766451365Z     at buildSshConfig (file:///app/dist/backend/backend/ssh/server-stats.js:1262:19)
[termix] 2026-02-09T20:12:47.766452166Z     at file:///app/dist/backend/backend/ssh/server-stats.js:274:28
[termix] 2026-02-09T20:12:47.766452857Z     at new Promise (<anonymous>)
[termix] 2026-02-09T20:12:47.766453899Z     at SSHConnectionPool.createConnection (file:///app/dist/backend/backend/ssh/server-stats.js:273:16)
[termix] 2026-02-09T20:12:47.766454811Z     at SSHConnectionPool.getConnection (file:///app/dist/backend/backend/ssh/server-stats.js:246:39)
[termix] 2026-02-09T20:12:47.766455602Z     at withSshConnection (file:///app/dist/backend/backend/ssh/server-stats.js:1273:41)
[termix] 2026-02-09T20:12:47.766456294Z     at file:///app/dist/backend/backend/ssh/server-stats.js:1367:30
[termix] 2026-02-09T20:12:47.766456945Z     at wrappedRequest (file:///app/dist/backend/backend/ssh/server-stats.js:490:25)
[termix] 2026-02-09T20:12:47.766457617Z     at RequestQueue.processQueue (file:///app/dist/backend/backend/ssh/server-stats.js:514:27)
[termix] 2026-02-09T20:12:47.766458287Z     at file:///app/dist/backend/backend/ssh/server-stats.js:502:18
[termix] 2026-02-09T20:12:48.085638991Z Configuring web UI to run on port: 8080
[termix] 2026-02-09T20:12:48.085652757Z SSL disabled - using HTTP-only configuration (default)
[termix] 2026-02-09T20:12:48.087150357Z Starting nginx...
[termix] 2026-02-09T20:12:48.090388304Z Starting backend services...`

### How to Reproduce

1. Create invalid format SSH keypair credential
2. Apply credential to server
3. Restart Termix

### Additional Context

_No response_

--- a/Casks/termix.rb
+++ b/Casks/termix.rb
@@ -1,6 +1,6 @@
 cask "termix" do
   version "2.0.0"
-  sha256 "a752ad4f05b4991b8a2ef986da80a86b79a70c93fe1045c4399fcf348a28c1d0"
+  sha256 "2e381ac504093bfa72d16ee20043640724394729b2ca0e6b26c9eac19aeb6d08"
 
   url "https://github.com/Termix-SSH/Termix/releases/download/release-#{version}-tag/termix_macos_universal_dmg.dmg"
   name "Termix"

--- a/src/backend/ssh/server-stats.ts
+++ b/src/backend/ssh/server-stats.ts
@@ -865,7 +865,13 @@ class PollingManager {
           latestConfig.statsConfig.metricsEnabled &&
           supportsMetrics(latestConfig.host)
         ) {
-          this.pollHostMetrics(latestConfig.host, latestConfig.viewerUserId);
+          this.pollHostMetrics(latestConfig.host, latestConfig.viewerUserId)
+            .catch((err) => {
+              statsLogger.error("Metrics polling failed", err, {
+                operation: "metrics_poll_unhandled",
+                hostId: host.id,
+              });
+            });
         }
       }, intervalMs);
     } else {
@@ -1913,7 +1919,8 @@ async function collectMetrics(host: SSHHostWithCredentials): Promise<{
         } else if (
           error.message.includes("No password available") ||
           error.message.includes("Unsupported authentication type") ||
-          error.message.includes("No SSH key available")
+          error.message.includes("No SSH key available") ||
+          error.message.includes("Invalid SSH key format")
         ) {
           authFailureTracker.recordFailure(host.id, "AUTH", true);
         } else if (

